### PR TITLE
Fixed issues with captions of certain tutorials

### LINKED ISSUE #529: Issue with captions of some tutorials
The captions of the following tutorials either read **TODO** or **Create TODO**:

https://ncas-cms.github.io/cf-python/tutorial.html#subspaces-in-time
https://ncas-cms.github.io/cf-python/tutorial.html#subspace-mode
https://ncas-cms.github.io/cf-python/tutorial.html#multiple-dimensions
https://ncas-cms.github.io/cf-python/tutorial.html#assignment-by-metadata
https://ncas-cms.github.io/cf-python/tutorial.html#encapsulating-conditions
https://ncas-cms.github.io/cf-python/tutorial.html#compound-conditions

Should they be like that?

### PR DESCRIPTION
**Pull request by bewithankit**
Fixes #529

### CODE REVIEW

File: docs/source/tutorial.rst [hunk 1]
```diff
@@ -3213,7 +3213,7 @@ type of subspace:
   unselected locations within the envelope.
 
   .. code-block:: python
-     :caption: *Create TODO*
+     :caption: *Create subspace by selecting the 1st, 2nd, 4th and 6th elements of the 'X' axis by creating an envelope of these elements and mising data within the envelope wherever needed.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3216]:
```suggestion
     :caption: *Create a subspace by selecting the 2nd, 3rd, 5th and 7th elements of the 'X' axis by creating an envelope of these elements with missing data within the envelope wherever needed.*
```

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -3238,7 +3238,7 @@ type of subspace:
   locations.
 
   .. code-block:: python
-     :caption: *Create TODO*
+     :caption: *Create subspace by selecting the 1st, 2nd, 4th and 6th elements of the 'X' axis with domain encompassing that of the original field contruct and mising data within the domain wherever needed.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3241]:
```suggestion
     :caption: *Create a subspace by selecting the 2nd, 3rd, 5th and 7th elements of the 'X' axis with domain encompassing that of the original field construct with missing data within the domain wherever needed.*
```

File: docs/source/tutorial.rst [hunk 3]
```diff
@@ -3272,7 +3272,7 @@ Conditions may also be applied to multi-dimensional metadata
 constructs
 
 .. code-block:: python
-   :caption: *Create TODO*
+   :caption: *Create subspace whose domain spans latitudes within the range of 51 to 53 degrees north, with the other domain axes remaining unchanged.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3275]:
```suggestion
   :caption: *Create a subspace whose domain spans latitudes within the range of 51 to 53 degrees north, with the other domain axes remaining unchanged.*
```

File: docs/source/tutorial.rst [hunk 4]
```diff
@@ -3606,7 +3612,7 @@ Multiple conditions may be combined with the Python bitwise "and"
 object.
 
 .. code-block:: python
-   :caption: *TODO*
+   :caption: *An example evaluating compound query operation involving a greater than or equal to 3 and a strictly less than 5 condition.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3615]:
```suggestion
   :caption: *An example evaluating a compound query operation involving a greater than or equal to 3 condition and a strictly less than 5 condition.*
```

File: docs/source/tutorial.rst [hunk 5]
```diff
@@ -3177,7 +3177,7 @@ type of subspace:
   are removed to create the returned subspace:
 
   .. code-block:: python
-     :caption: *Create TODO*
+     :caption: *Create subspace by compressing the domain spanning the 1st, 2nd, 4th and 6th elements of the 'X' axis, with the other domain axes remaining unchanged.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3180]:
```suggestion
     :caption: *Create a subspace by compressing the domain spanning the 2nd, 3rd, 5th and 7th elements of the 'X' axis, with the other domain axes remaining unchanged.*
```

File: docs/source/tutorial.rst [hunk 6]
```diff
@@ -3111,7 +3111,7 @@ Subspaces based on time dimensions may be defined with as
 date-time objects.
 
 .. code-block:: python
-   :caption: *TODO*
+   :caption: *Create subspaces in different ways based on the time dimension by selecting a particular date and using date-time and its queries.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3114]:
```suggestion
   :caption: *Create subspaces in different ways based on the time dimension by selecting a particular date and using date-time and date-time queries.*
```

File: docs/source/tutorial.rst [hunk 7]
```diff
@@ -3563,7 +3569,7 @@ or the Python equality (`==`) operator, the condition is evaluated in
 the context of that object.
 
 .. code-block:: python
-   :caption: *TODO*
+   :caption: *An example evaluating a strictly less than 3 query operation.*
```
Comment (davidhassell) [inline docs/source/tutorial.rst:3572]:
```suggestion
   :caption: *An example evaluating a strictly less than 3 query condition.*
```

**Review by davidhassell — CHANGES_REQUESTED** [review]
Hi Ankit - thank you for spotting these defects and fixing them!

I've made a few suggestions for you to consider - the main thing being the ordinal numbers were out by 1 :)

**Comment by bewithankit** [comment]
Ah, the ordinal numbers! Thank you David!

**Pull request merged**